locomotion-sdk/src/services/sharedStyles.js

```diff
@@ -0,0 +1,12 @@
+export const ERROR_COLOR = '#f35657';
+
+export const OPERATION_COLOR = '#24aaf2';
+
+export const FONT_STYLES = {
+  SIZE: {
+    HEADER_1: '20px',
```

Comment:
set
small
medium
large
extra_large

Reply:
but what if we use sizes in between? i switched it to names so we could leave spots for potentially other sizes. what would you call 14px if small and medium are already there?